### ISSUE TITLE
NVDA not reading updated state of checkbox

### ISSUE THREAD

**Issue by hvinh** [nvaccess/nvda#7165]
Hello,

There is an issue when we select a price cell: there is a checkbox behind but after checking it, NVDA does not announce the new state "checked".

This is working fine with other screen readers such as JAWS.

Page for testing:
https://tinyurl.com/lulax94

Could you please investigate?

Thanks & regards

**Comment by jcsteh** [comment]
This is a duplicate of #7136, which you also filed (and which I responded to).
